unit_tests/freshclam_test.py

```diff
@@ -263,8 +263,8 @@ def test_freshclam_05_cdiff_update(self):
         # start with this CVD
         shutil.copy(TC.path_source / 'unit_tests' / 'test_db' / 'test-1.cvd', TC.path_db / 'test.cvd')
 
-        # update to this CVD
```

Comment:
It appears that your commit somehow includes bits of a previous commit, I think as a result of a merge.  Can I trouble you to remove the changes to freshclam_test.py?

Reply:
Oops - should be fixed now.